keps/sig-node/4205-psi-metric/kep.yaml

```diff
@@ -22,17 +22,18 @@ see-also: []
 replaces: []
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: beta
+stage: stable
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.34"
+latest-milestone: "v1.35"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.33"
   beta: "v1.34"
+  stable: "v1.36"
```

Comment:
I wonder if we should leave the milestone unchanged. We can update it during the 1.36 cycle. When we update we should confirm that all GA criteria are met. 

Either way we will need to update the `latest-milestone: "v1.35"` line when 1.36 comes.